xarray/core/rolling_exp.py

```diff
@@ -99,17 +100,17 @@ def __init__(
         window_type: str = "span",
         min_weight: float = 0.0,
     ):
-        if has_numbagg is False:
+        if _NUMBAGG_VERSION is None:
             raise ImportError(
                 "numbagg >= 0.2.1 is required for rolling_exp but currently numbagg is not installed"
             )
-        elif has_numbagg < "0.2.1":
```

Comment:
Shall we just require a minimum of `0.3.1`?

Reply:
I had this originally, but we have an automated check for versions available 12 months ago. And the convention makes sense to me, so I thought this was OK.

I'm also adding lots of other methods; e.g. https://github.com/pydata/xarray/pull/8307, so we're going to have some months of version proliferation regardless...